libraries/Microsoft.Bot.Builder.AI.QnA/QnAMaker.cs

```diff
@@ -216,14 +216,14 @@ public Task<QueryResult[]> GetAnswersAsync(ITurnContext turnContext, QnAMakerOpt
 
             if (string.IsNullOrEmpty(turnContext.Activity.Text))
             {
-                throw new ArgumentException("Null or empty text");
+                throw new ArgumentException("Question cannot be null or empty text");
             }
 
-            var result = await this._generateAnswerHelper.GetAnswersRawAsync(turnContext, messageActivity, options).ConfigureAwait(false);
+            QueryResults results = await this._generateAnswerHelper.GetAnswersRawAsync(turnContext, messageActivity, options).ConfigureAwait(false);
```

Comment:
Use `var` instead of explicit types when possible
```suggestion
            var results = await this._generateAnswerHelper.GetAnswersRawAsync(turnContext, messageActivity, options).ConfigureAwait(false);
```